plugins/module_utils/netbox_utils.py

```diff
@@ -210,6 +226,12 @@
     "nat_inside": set(["vrf", "address"]),
     "parent_region": set(["slug"]),
     "platform": set(["slug"]),
+    "power_feed": set(["name"]),
```

Comment:
It appears that `power_panel` is requried in the `netbox_power_feed` module so I would probably add that as an `ALLOWED_QUERY_PARAM` for `power_feed` and any others for the other modules since you're requiring those.

Reply:
Added, can you take a look if the new settings are right?

Reply:
They look correct to me. I will go ahead and merge this PR.

Thank you so much for these PRs you have submitted! They were really high quality and I appreciate that!

I think the docs may be a bit out of date, but were they helpful?
Was it easy to make these new modules?
Would you change anything about the docs or contribution process?

Looking for honest feedback so don't be scared to be honest :)

Reply:
> Thank you so much for these PRs you have submitted! They were really high quality and I appreciate that!

Thanks for the kind words! :)
I am going to add a few more modules next week.

> I think the docs may be a bit out of date, but were they helpful?

They were really helpful! Without the docs it would have been much more complicated.

> Was it easy to make these new modules?

The first one was a bit tricky. For the modules afterwards the biggest problem is doing clean work and not being careless.

> Would you change anything about the docs or contribution process?
While the documentation is already very good I would probably structure it a little bit different.
I will have a look into it next week, after finishing the other modules.
The biggest pain points are probably the unorderly structure (at least for my think process) and implicit knowledge which is not available to beginners.